h2o-py/tests/testdir_apis/Data_Manipulation/pyunit_h2oH2OFrame_columns_by_type.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import print_function
+import sys
+sys.path.insert(1,"../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.utils.typechecks import assert_is_type
+
+def h2o_H2OFrame_columns_by_type():
+    """
+    Python API test: h2o.frame.H2OFrame.columns_by_type(coltype='numeric')
+
+    Copied from pyunit_colnames_by_type.py
+    """
+    fr = h2o.import_file(pyunit_utils.locate("smalldata/jira/filter_type.csv"))
+
+    #Positive case. Look for all coltypes
+    num_type = fr.columns_by_type() #numeric by default
+    cat_type = fr.columns_by_type(coltype="categorical")
+    str_type = fr.columns_by_type(coltype="string")
+    time_type = fr.columns_by_type(coltype="time")
+    uuid_type = fr.columns_by_type(coltype="uuid")
+    bad_type = fr.columns_by_type(coltype="bad")
```

Comment:
see if can be merged with pyunit_colnames_by_type

Reply:
There is no such command...